packages/components/footers/Footer.tsx

```diff
@@ -16,75 +16,78 @@ import logoSVG from "../../../assets/logos/logo.svg";
 import { secondaryColor } from "../../utils/style/colors";
 import { layout } from "../../utils/style/layout";
 import { SVG } from "../SVG";
-import { LegacyTertiaryBox } from "../boxes/LegacyTertiaryBox";
+import { TertiaryBox } from "../boxes/TertiaryBox";
 
 const FooterSocialNetworks: React.FC = () => {
   return (
-    <LegacyTertiaryBox
-      mainContainerStyle={{ padding: layout.spacing_x1, flexDirection: "row" }}
-      style={{ marginBottom: layout.contentSpacing }}
+    <TertiaryBox
+      style={{
+        marginBottom: layout.contentSpacing,
+        padding: layout.spacing_x1,
+        flexDirection: "row",
+      }}
     >
       <TouchableOpacity
         style={{ marginRight: layout.spacing_x1 }}
         onPress={() =>
           Linking.openURL("https://www.coingecko.com/en/coins/teritori")
         }
       >
-        <LegacyTertiaryBox
-          mainContainerStyle={{
+        <TertiaryBox
+          style={{
             borderColor: secondaryColor,
             borderRadius: 12,
             padding: layout.spacing_x1_5,
```

Comment:
Maybe we can extract these lines in a variable because we use this many times in this file ? (you can find some example here https://github.com/TERITORI/teritori-dapp/blob/main/packages/components/music/FeedMusicList.tsx)